hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/CompactionRequestImpl.java

```diff
@@ -149,6 +158,7 @@ public int hashCode() {
     result = prime * result + ((storeName == null) ? 0 : storeName.hashCode());
     result = prime * result + (int) (totalSize ^ (totalSize >>> 32));
     result = prime * result + ((tracker == null) ? 0 : tracker.hashCode());
+    result = prime * result + (isAfterSplit ? 1231 : 1237);
```

Comment:
This pattern is not one I've seen before but we do it above for `isOffPeak` so (shrug).

Reply:
Yeah right, I exactly looked for an existing boolean and applied same formula rather than regenerating hashcode() by IDE :)